doc/source/migration.rst

```diff
@@ -56,12 +56,13 @@ Migration to v3 version
       fig, ax = matplotlib_utils.get_figure()
       matplotlib_impl.plot_dendrogram(m, ax)
       matplotlib_utils.plot_style(fig=fig, ax=ax)
-      matplotlib_utils.save_plot(fig=fig, output_path=output_path)
+      matplotlib_utils.save_plot(fig=fig, output_path="output-directory-path")
```

Comment:
Any ideas on how to nicely prevent this directory creation on the root level after doctest runs?

Using tempfile here would make this example more obscure, wouldn't it?

Reply:
I could comment this out as well.